samples/routing/flowcontrol/Core_6/Client/Program.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Configuration.AdvanceExtensibility;
+using NServiceBus.Logging;
+using NServiceBus.Routing;
+
+class Program
+{
+
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.FlowControl.Client";
+        LogManager.Use<DefaultFactory>()
+            .Level(LogLevel.Info);
+
+        const string letters = "ABCDEFGHIJKLMNOPQRSTUVXYZ";
+        var random = new Random();
+
+        var endpointConfiguration = new EndpointConfiguration("Samples.FlowControl.Client");
+        endpointConfiguration.UseSerialization<JsonSerializer>();
+        endpointConfiguration.UsePersistence<InMemoryPersistence>();
+        endpointConfiguration.EnableInstallers();
+        endpointConfiguration.SendFailedMessagesTo("error");
+
+        #region Routing
+        endpointConfiguration.UnicastRouting().RouteToEndpoint(typeof(PlaceOrder), "Samples.FlowControl.Server");
+        var server = new EndpointName("Samples.FlowControl.Server");
+        endpointConfiguration.UnicastRouting().Mapping.Physical.Add(server,
```

Comment:
that would no longer be possible with the new API proposal. Would you use the instance mapping file instead?